backend/typescript/middlewares/validators/authValidators.ts

```diff
@@ -7,11 +7,17 @@ export const loginRequestValidator = async (
   res: Response,
   next: NextFunction,
 ) => {
-  if (!validatePrimitive(req.body.email, "string")) {
-    return res.status(400).send(getApiValidationError("email", "string"));
-  }
-  if (!validatePrimitive(req.body.password, "string")) {
-    return res.status(400).send(getApiValidationError("password", "string"));
+  if (req.body.id_token) {
```

Comment:
should be idToken, we only change it to id_token for python in the baseAPIClient interceptor